src/analysis/normed_space/basic.lean

```diff
@@ -448,7 +454,7 @@ by { convert tendsto_iff_dist_tendsto_zero, simp [dist_eq_norm] }
 
 lemma tendsto_zero_iff_norm_tendsto_zero {f : γ → β} {a : filter γ} :
   tendsto f a (𝓝 0) ↔ tendsto (λ e, ∥f e∥) a (𝓝 0) :=
-by simp [tendsto_iff_norm_tendsto_zero]
+by { rw [tendsto_iff_norm_tendsto_zero], simp only [sub_zero] }
```

Comment:
I don't understand why this `simp` stopped working. I fixed it here, but the build is already breaking in another place. And I didn't even touch any simp-lemmas.

Reply:
This is using a lemma that is not a good simp lemma:
```
0. [simplify.rewrite] [tendsto_iff_norm_tendsto_zero]: tendsto f a (𝓝 0) ==> tendsto (λ (e : γ), ∥f e - 0∥) a (𝓝 0)
0. [simplify.rewrite] [sub_zero]: f e - 0 ==> f e
0. [simplify.rewrite] [tendsto_iff_norm_tendsto_zero]: tendsto (λ (e : γ), ∥f e∥) a (𝓝 0) ==> tendsto (λ (e : γ), ∥∥f e∥ - 0∥) a (𝓝 0)
0. [simplify.rewrite] [sub_zero]: ∥f e∥ - 0 ==> ∥f e∥
0. [simplify.rewrite] [tendsto_iff_norm_tendsto_zero]: tendsto (λ (e : γ), ∥∥f e∥∥) a (𝓝 0) ==> tendsto (λ (e : γ), ∥∥∥f e∥∥ - 0∥) a (𝓝 0)
0. [simplify.rewrite] [sub_zero]: ∥∥f e∥∥ - 0 ==> ∥∥f e∥∥
0. [simplify.rewrite] [tendsto_iff_norm_tendsto_zero]: tendsto (λ (e : γ), ∥∥∥f e∥∥∥) a (𝓝 0) ==> tendsto (λ (e : γ), ∥∥∥∥f e∥∥∥ - 0∥) a (𝓝 0)
```

Reply:
I would suggest `by simpa using @tendsto_iff_norm_tendsto_zero _ _ _ f a 0` or `tendsto_iff_norm_tendsto_zero.trans (by simp)`

Reply:
@digama0 Do you understand why the build is breaking outside this file?